docs/guide/drivers/123.md

```diff
@@ -28,4 +28,10 @@ The password used to log in
 ### Root directory file_id
 Enter the folder you want to mount, the last string of the official website URL, such as:
 
-![123](/img/drivers/123.png)
\ No newline at end of file
+![123](/img/drivers/123.png)
+
+### Recommendations
+
+- It seems that the 123 API has a limited number of loads each time, so if you load hundreds of files in a folder at one time, an error may be reported
+- It is recommended to use paging loading: about 50 pages are loaded each time, which is not a big problem
+  - manage ---> settings ---> Site ---> Pagination type(choose by yourself) ---> Default page size(set it yourself) ---> Save
```

Comment:
> It's useless to do this, because paging is also done after all requests are made. A useful approach is not to put too many sub-items in the same folder.

It has been fixed, I don’t know why I didn’t receive the email notification, it’s late